.travis.yml

```diff
@@ -53,6 +54,8 @@ install:
   - unzip -q sdk-tools-linux-$ANDROID_SDK_VERSION.zip -d $ANDROID_HOME
   - export SDKMANAGER_OPTS="--add-modules java.xml.bind"
   - yes | $ANDROID_HOME/tools/bin/sdkmanager --verbose "platform-tools"
+  - curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain $RUST_VERSION
```

Comment:
This line looks more complex than the one at https://forge.rust-lang.org/other-installation-methods.html, why?

Reply:
This the official line from https://rustup.rs/ which is more secure, since it allows only tlsv1.2 https connections.